### ISSUE TITLE
Building options

### ISSUE THREAD

**Issue by bradders** [fourstacks/nova-checkboxes#3]
What's the best/easiest way to build an `->options([])` with external data?

I was hoping to pass in `Region::all()` to populate my options, possible?

**Comment by bradders** [comment]
Pretty sure this isn't an issue, per se, I was just wondering whether there was a way to throw a collection at the `make()` call. I've loaded the values elsewhere, although I've now ran into the issue mentioned here: https://github.com/fourstacks/nova-checkboxes/issues/2

**Comment by fourstacks** [comment]
Hi @bradders - I'm guessing you have a model called Region and that you are just using the eloquent `all()` method on it?  If that is the case then you could certainly use collection methods to populate the options for this field.  I don't know what fields your Region model has so this is just pseudo code but something like this should work:

```
Checkboxes::make('Selected Regions')
    ->options(
        Region:all()
            ->mapWithKeys(function($region){
                return [$region['slug'] => $region['name']]; 
            })
            ->all()
    )
```
That said, if you are planning on using the ID of the Region as the option values then I think you might run into the issue raised in #2.  I'm working on a fix for that and should have something shipped soon

**Comment by bradders** [comment]
Yeah, exactly that! Thanks for the code sample, I'd achieved it in a similar fashion. But you're right, I'm looking to key by ID so the fix for #2 will be greatly received!
